PEM-7063 Imported cluster limitations (#6376)
spectrocloud/librarium: docs/docs-content/clusters/imported-clusters/imported-clusters.md

| -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Full Cluster Profile usage | You cannot use a full cluster profile. You are limited to using add-on profiles when deploying cluster profiles to imported clusters. This does not include full cluster profiles that are specifically created for [EKS Hybrid node pools](../public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md). |
| Kubeconfig file access     | You cannot download the cluster's kubeconfig file from Palette. You must use the underlying infrastructure provider to access the kubeconfig file.                                                                                                                                                             |

> | HTTP Proxy Configuration   | You cannot configure HTTP proxy settings for imported clusters with read-only mode. Import the cluster using Full Permission mode.                                                                                                                                                                             |

Reviewer: Is the _Full Permission mode_ an alternative for _read-only mode_? If so, shouldn't the _read-only mode_ be capitalized as well?

<br />

### Generic Clusters

- Palette displays limited metadata for imported generic clusters when compared with cloud-specific clusters. The
  metadata displayed for generic clusters are `region` and `instance type`.

### Cloud-Specific Clusters

Imported Cloud-specific clusters provide a similar experience as Palette deployed clusters but with the following
limitations:

- Palette cannot manage [node groups/pools](../cluster-management/node-pool.md) for imported cloud-specific clusters
  because it does not provision the nodes directly.

  - This does not apply to

>     [Amazon EKS Hybrid node pools](../public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md) as they are
>     comprised of edge hosts that are managed by Palette.

Reviewer: ```suggestion
    [Amazon EKS Hybrid node pools](../public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md), which consist of edge hosts managed by Palette.
```

Maybe like this (to avoid passive voice)?

Author: I like it.

> - When using the Palette API to interact with an imported Amazon EKS, GKE, or AKS cluster, the IaaS endpoint must be
>   used. Refer to the following tabs for examples.

Reviewer: ```suggestion
- When interacting with imported EKS, GKE, or AKS clusters through the Palette API, use the IaaS endpoint. Refer to the following tabs for examples.
```
